tools/astarte_e2e/lib/astarte_e2e_web/email.ex

```diff
@@ -0,0 +1,42 @@
+#
+# This file is part of Astarte.
+#
+# Copyright 2020 Ispirata Srl
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+defmodule AstarteE2EWeb.Email do
```

Comment:
In Cloud Manager this was in the Web namespace because it was used by Web stuff, but in this case the module should probably live under the `service_notifier` context, i.e. `AstarteE2E.ServiceNotifier.Email` (in `lib/astarte_e2e/service_notifier/email.ex`)